src/schemas/SettingsSchema.js

```diff
@@ -13,6 +13,7 @@ module.exports = graphql.buildSchema(`
   type Mutation {
     createSite(input: SiteDefinition!): Site
     removeSite(input: SiteDefinition!): Site
+    createStream(input: StreamDefinition!): Stream
```

Comment:
Could this return a StreamDefinition so that you don't need to have both types around?